packages/data-portal-table/src/components/DataTableControls.tsx

```diff
@@ -21,6 +21,8 @@ interface IDataTableControlsProps {
     searchBoxPlaceHolder?: string;
     filterText?: string;
     extraControls?: JSX.Element;
+    downloadButtonLabel?: string;
+    showColumnSelect?: boolean;
```

Comment:
Let's have a `hideColumnSelect` option instead because we show `ColumnSelect` by default

```suggestion
    hideColumnSelect?: boolean;
```